client/app/components/calendar/event/EventDetails.jsx

```diff
@@ -1,30 +1,54 @@
+import flow from 'lodash/fp/flow';
+import filter from 'lodash/fp/filter';
+import map from 'lodash/fp/map';
 import React from 'react';
 import { eventTimeString } from 'helpers/DateHelper';
 import EventSchema from 'schemas/EventSchema';
+import { If } from 'react-if';
+
 import './event.scss';
 
-export default class EventDetails extends React.Component {
-  static propTypes = {
-    event: EventSchema.isRequired,
-    timeFormat: React.PropTypes.string
-  };
+const defaultCreator = { display_name: 'private', self: false };
+const creator = (event) => event.creator || defaultCreator;
 
-  render() {
-    const undefinedCreator = { display_name: 'private', self: false };
-    const creator = this.props.event.creator || undefinedCreator;
-    return (
-      <div>
-        <div className="event-time">{eventTimeString(this.props.event, this.props.timeFormat)}</div>
-        <div className="event-name">{this.props.event.summary}</div>
-        <div className="event-user">
-          <small>by&nbsp;</small>
-          {creator.display_name || creator.email}
-        </div>
-        <div className="event-location">
-          <small>in&nbsp;</small>
-          {this.props.event.conference_room.title}
-        </div>
+const EventDetails = ({ event, timeFormat, showLocation, showGuests }) => (
```

Comment:
Can you check why github failed to provide us with fancy js colors for some files?